pkg/controllers/jobset_controller.go

```diff
@@ -731,7 +733,7 @@ func labelAndAnnotateObject(obj metav1.Object, js *jobset.JobSet, rjob *jobset.R
 	jobName := placement.GenJobName(js.Name, rjob.Name, jobIdx)
 
 	// Set labels on the object.
-	labels := collections.CloneMap(obj.GetLabels())
+	labels := make(map[string]string)
```

Comment:
this caused a regression, see https://github.com/kubernetes-sigs/jobset/issues/729